src/Nri/Ui/Alert/V4.elm

```diff
@@ -0,0 +1,226 @@
+module Nri.Ui.Alert.V4 exposing
+    ( error, success, tip, warning
+    , complexError, complexWarning
+    , somethingWentWrong
+    )
+
+{-|
+
+
+# Changes from V3:
+
+  - Changes the error font color from `purpleDark` to `purple`
+  - Adds `complexError` and `complexWarning` errors that support Html arguments
+  - Adds `somethingWentWrong` error for displaying stacktraces
+
+
+# About:
+
+UI components that highlight information to the user.
+
+@docs error, success, tip, warning
+@docs complexError, complexWarning
+@docs somethingWentWrong
+
+-}
+
+import Accessibility.Styled as Html exposing (Html)
+import Css
+import Css.Global
+import Html.Styled exposing (fromUnstyled)
+import Html.Styled.Attributes exposing (css)
+import Markdown
+import Nri.Ui
+import Nri.Ui.Colors.V1 as Colors
+import Nri.Ui.Fonts.V1 as Fonts
+import Nri.Ui.Icon.V3 as Icon
+import Nri.Ui.SpriteSheet exposing (bulb, checkmark, exclamationMark)
+import Nri.Ui.Svg.V1 as NriSvg exposing (Svg)
+
+
+{-| -}
+complexError : Html msg -> Html msg
+complexError content =
+    alert
+        [ exclamation Colors.purple
+        , alertHtml Colors.purple [ content ]
+        ]
+
+
+{-| -}
+complexWarning : Html msg -> Html msg
```

Comment:
💭 should we just make `warning` and `error` take Html instead of String, since it could still be used with `warning (text "...")` ?  I guess I like having the one that directly takes a string, but is it worth having a much bigger API?

Reply:
I guess alternatively, why do we need alerts to be able to contain custom HTML?  Maybe an Alert is incorrectly being used in the places that's needed?  (Maybe could the styleguide example be updated to show an example of this that is an actual example of where custom HTML in the alert is needed?)

Reply:
Partly, there are a LOT of alert uses in the monolith already -- the API is already broad, so this work will streamline where the implementations live/forces to get the colors consistent (which they haven't) and get styles to match up and all.

I'll see if I can find some uses in the repo of the actual html alerts and use them in the exampels. I'm also not sure why we support markdown (do we actually use this?) but didn't want to check literally 80-plus pages 😅.

Reply:
Oh, I also deleted complexTip and complexSuccess from the monolith cause we weren't using them. I suspect that in the long run (maybe V5?) we'll delete the html support

Reply:
We had at least one alert that contained a link/javascript action (when editing a quick write's due date), but I think that should no longer be relevant (might not be actually be dead because feature flag). There may be others, not sure

Reply:
> Partly, there are a LOT of alert uses in the monolith already

Doesn't the module versioning we use solve this?  (Is there a reason you want all the callers to upgrade to V4 asap?)

Reply:
@avh4 So I converted a bunch of complex html alerts to markdown-based alerts in the monolith.
There's one html-based alert left:

<img width="820" alt="Screen Shot 2019-04-17 at 4 11 55 PM" src="https://user-images.githubusercontent.com/8811312/56326698-f72e3780-612b-11e9-8ab8-104e0647ba94.png">

I don't see a way around that one being html-based.

Buuut I think that you're right that the API doesn't need to support html. So I'll drop the complex views from this PR.

Reply:
oops, missed your comments.

So the reason that I want to upgrade everything is that prior to today, we weren't actually on the Nri.Ui.Alert version consistently -- we were using `Nri.Alert` and `Nri.AlertDEPRECATED` and `Nri.Ui.Alert.V2`. Also, the color of the alert changed between some of the versions, and I like for colors to match up 😅